content/bizops/index.md

```diff
@@ -62,6 +63,14 @@ A BizOps team member will triage and assign issues to the relevant team member.
 
 For _time-sensitive_ and _urgent_ requests, please tag the issue with the `urgent` label. This will make the GitHub issue appear in the #analytics channel in Slack to ensure visibility, where you can follow up with more details/asks.
 
+### What a great request looks like
+
+- **The problem:** We don't have a great high-level dashboard for the product/eng leadership team to get a pulse on our product KPIs.
+- **End state:** [PDF of a sketch](https://drive.google.com/file/d/1X2VKoWUvJpLS7LMMkG3bmhA7DJkXVujB/view) and a [doc of all the metrics](https://docs.google.com/document/d/111lB3rPDU3DHiiu4fc8MCVAK_-b20HL9cM4BkIkbmk8/edit#).
```

Comment:
Is that PDF public? I assumed not but it let me see a preview so I couldn't tell..

Reply:
Also could be useful to list some tools people use to do mockups (I'm actually not sure which tools people are using for this? Excalidraw and Miro presumably?)